daico/contracts/ICOScheme.sol

```diff
@@ -1,4 +1,4 @@
-pragma solidity ^0.4.25;
+pragma solidity ^0.4.24;
```

Comment:
why 24?

Reply:
I was running into an issue with truffle, so I tried different versions believing that might have been the difference. Basically `truffle migrate` kept throwing errors, so I tried downgrading the dependencies believing that might have been the cause to the errors. I'll re-add the original version as this doesn't seem to the cause of the issues. Currently still running into a "revert" issue every other time I run "truffle migrate" It usually goes away after I restart my machine. I've tried killing terminal and processes in the activity monitor but to no avail.

Reply:
The issue I'm currently working on, when I don't face the revert issue, is passing the avatar address from the deploy_dao.js. In the peepeth example the console.log called to get the avatar address is within a promise, wondering if this is why I'm not seeing the same address appear within the daico example.

Reply:
I'm not sure I quite understand what is the problem you are having with the Avatar. Can you please elaborate on that?

Reply:
So I need to capture the address of the avatar to pass it onto the index.js. However the console.log I have in the deploy.js don't return in the terminal nor in the google chrome developer console. I'm looking to capture the avatar address, but I'm not able to see it anywhere. I'm wondering if my console.log() calls are ever executed.